NEWS.md

```diff
@@ -33,6 +33,8 @@ changes:
   * Added an `openssl configutl` utility for processing the openssl
     configuration file and dumping the equal configuration file.
 
+  * ANSI-C is no longer required. See NOTES-C99.md for details.
```

Comment:
I think more correct would be `ANSI-C is no longer sufficient for building OpenSSL. See...` and I would probably repeat the same in CHANGES.md.

Reply:
I think the right phrasing here rests on if we mean ANSI-C (the language specification) vs ANSI-C (A toolchain compliant with at most that standard).  In the latter case I think the phrasing is:
```
A toolchain that supports at most the ANSI-C standard is no longer sufficient for building OpenSSL
```
whereas we could also state
```
Writing ANSI-C compliant code is no longer required.
```

Note sure which is the more clear statement.

Reply:
`Writing ANSI-C compliant code is no longer required.` - this does not belong into NEWS.md or CHANGES.md as that is supposed to be primarily read by users of OpenSSL.